thorlcr/master/thmastermain.cpp

```diff
@@ -636,14 +636,21 @@ int main( int argc, const char *argv[]  )
     ILogMsgHandler *logHandler;
     try
     {
+#ifndef _CONTAINERIZED
         {
             Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(globals, "thor");
             lf->setName("thormaster");//override default filename
             lf->setCreateAliasFile(false);
             logHandler = lf->beginLogging();
             createUNCFilename(lf->queryLogFileSpec(), logUrl, false);
+
+            LOG(MCdebugProgress, thorJob, "Opened log file %s", logUrl.str());
         }
-        LOG(MCdebugProgress, thorJob, "Opened log file %s", logUrl.str());
+#else
+        logHandler = queryStderrLogMsgHandler();
+        logUrl.set("stderr");
```

Comment:
@jakesmith please comment on this change, the logUrl is written to "globals[@logURL]" but it's unclear how that value is ultimately consumed and the generic "stderr" might not be appropriate.
Similar code applies to thslavemain below. 

BTW, in this change, we're removing the use of per component log file in container mode.

Reply:
@logUrl isn't used anywhere, must be legacy.
So feel free to delete it.

Reply:
Separate issue, but currently Thor (and other components) add the IP + path to the workunit via workunit::addProcess.
That's going to need to change.